templates/web/base/report/updates.html

```diff
@@ -1,11 +1,11 @@
 [% global.last_state = '' %]
-[% prev_state = '' %]
 [% FOREACH update IN updates %]
-[%- IF prev_state == 'hidden' OR prev_state == 'unconfirmed' OR update.problem_state == 'hidden' OR update.problem_state == 'unconfirmed' %]
-  [%- prev_state = update.problem_state %]
+[%- IF global.last_state == 'Hidden' OR global.last_state == 'Unconfirmed' OR update.problem_state == 'hidden' OR update.problem_state == 'unconfirmed' %]
+  [%- IF update.problem_state != '' %]
```

Comment:
It'd be more consistent if this did the same as `update.html` and checked the result of `problem_state_display` (not sure it makes any difference, it shouldn't, but might as well?).

[edit: Actually, then given my next comment about translating strings, perhaps `global.last_state` should be changed to use `problem_state` rather than problem state display, so that translation is not necessary. But meh]

Reply:
👍